resources/js/visual-editor/blocks/embed/embed.css

```
@@ -0,0 +1,156 @@
+}
+
+.wp-block-embed {
+    overflow-wrap: break-word;
+    margin: 0 0 1em 0;
+}
+
+/* Inlined caption-style() + caption-style-theme() mixins. */
+.wp-block-embed :where(figcaption) {
+    margin-top: 0.5em;
+    margin-bottom: 1em;
+    color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)